packages/gasket-plugin-cypress/lib/index.js

```diff
@@ -0,0 +1,55 @@
+const { devDependencies, name } = require('../package.json');
+
+module.exports = {
+  name,
+  hooks: {
+    create: {
+      timing: {
+        last: true,
```

Comment:
Should `last` be here if it goes before `@gasket/plugin-lint`? There may be a conflict or undefined behavior.

Reply:
I'm curious how you ended up with this timing setting as well. I'm going to guess you just need to make sure this runs _after_ the `@gasket/plugin-nextjs` to check if React has been added or not, correct?

In which case, this should work for you:
```
timing: {
  after: ['@gasket/plugin-nextjs']
}
```

Reply:
Okay! I saw that both`@gasket/plugin-mocha` and `@gasket/plugin-jest` had that setup so I used that for inspiration. I'll make that change to the timing!

Reply:
Ah, you are correct. Ok, it may be a suitable approach to leave as is then.